File: Modul-2/Soal-4/PRAK204-2410817310012-FAISALTANJUNG.c
#include<stdio.h>
#include <math.h>

#define pi 22/7

int main(){

    int jari_jari, tinggi;
    float keliling, luas, volume;

    printf(" ");
    scanf("%d %d", &jari_jari, &tinggi);
    
    volume =(float) 22 / 7 * (jari_jari * jari_jari) * tinggi;
    luas =(float) 2 * 22 / 7 * jari_jari * (jari_jari + tinggi);
    keliling =(float)2 * 22 / 7 * jari_jari;

    printf("Volume = %.2f \nLuas = %.2f \nKeliling = %.2f", volume, luas, keliling);
    return 0;
}

File: Modul-2/Soal-4/PRAK204-2410817310012-FAISALTANJUNG.py
total_variabel = []
while len(total_variabel) < 2:
    line = input()
    if line :
        total_variabel.extend(map(float, line.split()))

jari_jari, tinggi = total_variabel[:6]

volume = 22 / 7 * (jari_jari * jari_jari) * tinggi
luas = 2 * 22 / 7 * jari_jari * (jari_jari + tinggi)
keliling = 2 * 22 / 7 * jari_jari

print(f"Volume = {volume:.2f} \nLuas = {luas:.2f} \nKeliling = {keliling:.2f}\n")